cmd/ci-reporter/cmd/testgrid.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// RequiredTestgridJob specifies a testgrid job like 'Master-Blocking'
+type RequiredTestgridJob struct {
+	OutputName string
+	URLName    string
+}
+
+// GetTestgridReportData used to request the raw report data from testgrid
+func GetTestgridReportData(cfg ReporterConfig) ([]TestgridStatistics, error) {
+	requiredJobs := []RequiredTestgridJob{
+		{OutputName: "Master-Blocking", URLName: "sig-release-master-blocking"},
+		{OutputName: "Master-Informing", URLName: "sig-release-master-informing"},
+	}
+
+	if cfg.ReleaseVersion != "" {
+		requiredJobs = append(requiredJobs, []RequiredTestgridJob{
+			{OutputName: cfg.ReleaseVersion + "-blocking", URLName: "sig-release-" + cfg.ReleaseVersion + "-blocking"},
+			{OutputName: cfg.ReleaseVersion + "-informing", URLName: "sig-release-" + cfg.ReleaseVersion + "-informing"},
+		}...)
+	}
```

Comment:
I think all of this can be simplified by setting `cfg.ReleaseVersion` to a default value of `master` and then doing:

```suggestion
		requiredJobs = append(requiredJobs, []RequiredTestgridJob{
			{OutputName: cfg.ReleaseVersion + "-blocking", URLName: "sig-release-" + cfg.ReleaseVersion + "-blocking"},
			{OutputName: cfg.ReleaseVersion + "-informing", URLName: "sig-release-" + cfg.ReleaseVersion + "-informing"},
		}...)
```

Reply:
Since we always like to see master blocking / master informing we can not further simplify this. But thanks for the thought - we can discuss if this would be a better way to generate testgrid reports